src/components/Editor/Preview/Popup.js

```diff
@@ -85,25 +86,27 @@ export class Preview extends Component {
 
     return (
       <div
-        className="preview"
+        className="preview-popup"
         onClick={() => selectSourceURL(location.url, { line: location.line })}
       >
-        {previewFunction(value)}
+        {true
```

Comment:
this will always return `previewFunction(value)`; maybe we should just use it directly?